indico/modules/attachments/controllers/event_package.py

```diff
@@ -67,9 +69,25 @@ def adjust_path_length(segments):
 def _get_start_dt(obj):
     # TODO: adapt to new models (needs extra properties to use event TZ)
     if isinstance(obj, SubContribution):
-        return obj.getContribution().getAdjustedStartDate()
-    else:
-        return obj.getAdjustedStartDate()
+        return obj.contribution.timetable_entry.start_dt if obj.contribution.timetable_entry else None
+    elif isinstance(obj, Contribution):
+        return obj.timetable_entry.start_dt if obj.timetable_entry else None
+    elif isinstance(obj, Session):
+        return
+    return obj.start_dt
+
+
+def _get_obj_parent(obj):
+    if isinstance(obj, SubContribution):
+        return obj.contribution
+    elif isinstance(obj, Contribution):
+        if obj.session:
+            return obj.sesion
+        elif obj.session_block:
+            return obj.session_block
```

Comment:
I don't think you ever want the session block here (the old getOwner() didn't include session blocks either)